react/__tests__/OrderBy.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+import React from 'react'
+import { render } from 'test-utils'
+
+import OrderBy from '../OrderBy'
+import { setMobileState } from 'vtex.render-runtime'
+
+describe('<OrderBy />', () => {
+  const renderComponent = customProps => mobile => {
+    setMobileState(mobile)
+    const props = {
+      getLinkProps: () => ({ page: 'mokedPage' }),
+      orderBy: 'OrderByTopSaleDESC',
+      ...customProps,
+    }
+
+    return render(<OrderBy {...props} />)
+  }
+
+  it('should match snapshot in web mod', () => {
+    const { asFragment } = renderComponent()(false)
+
+    expect(asFragment()).toMatchSnapshot()
+  })
+
+  it('should match snapshot in mobile', () => {
+    const { asFragment } = renderComponent()(true)
```

Comment:
You need to add a afterEach() setting the mobile to the default value (false).
Because depending in the order that other tests in this app run, they could change because se mobile value is `true`